imports/api/parties/parties_collection.js

```diff
@@ -30,13 +30,13 @@ export const Parties = new scorpius.collection('parties', {
 
 Parties.allow({
   insert: function (userId, doc) {
-    return (userId && doc.userId === userId);
+    return (userId && doc.userId === userId) || Meteor.user().hasRole("moderator");
```

Comment:
In that case, do we not need to explicitly allow admin in any place where we allow mods to act? Or alternatively, should we automatically add the moderator role to all admin users?